app/views/groups/samples/_table.html.erb

```diff
@@ -41,6 +41,27 @@
           field: :name
         ) %>
       </th>
+      <th
```

Comment:
**chore:** For all the tables, can you make the ID (puid) the first column this will now become the fixed column for all tables.

Reply:
Fixed in 731734448cf99512fac5eba96ea2260c5183b13e